jenkins/gradle/gradle-check.jenkinsfile

```diff
@@ -79,6 +79,13 @@ pipeline {
         BUILD_CAUSE = currentBuild.getBuildCauses()
     }
     stages {
+        stage('Check and abort stale runs') {
+            steps {
+                script {
+                    abortStaleJenkinsJobs(jobName: 'gradle-check', lookupTime: 3)
```

Comment:
Just thinking out loud, do you think there will be an edge case scenario that the PR author/user wants to continue running gradle check for previous commit?
I don't think it would matter anyway because the GHA check would consider the most recent run right?

Reply:
I don't see any reason to keep an old job running if you have pushed a new commit. Only the result on the new run matters for approval.